frontend/components/ui/Typography/Title5.tsx

```diff
@@ -0,0 +1,22 @@
+import { Typography } from 'antd';
+
+const { Title } = Typography;
+
+export const Title5 = ({
+  children,
+  ...props
+}: {
+  children: React.ReactNode;
+} & React.ComponentProps<typeof Title>) => {
+  const { style, className, ...rest } = props;
+  return (
+    <Title
+      className={`m-0 ${className}`}
+      level={5}
+      style={{ fontSize: 20, lineHeight: 1.4, fontWeight: 500, ...style }}
```

Comment:
can't we handle styles for different typography titles via theme tokens? we have many places where antd Title is used, and I'm a bit concerned we now have two different approaches 
<img width="311" height="744" alt="image" src="https://github.com/user-attachments/assets/996a0278-1ce2-46ef-96e2-0a709968ede8" />

Reply:
also could maybe rename Title5 -> H5? wdym, it feels more common to me, but might be wrong

Reply:
I think it's okay, the whole app is revised by this redesign. @mohandast52 wdyt?